### DISCUSSION TITLE
Visualizing Transformer based on your notebook

### DISCUSSION THREAD

**Discussion by jackli777** [rasbt/machine-learning-book#86] [Show and tell]
Dear Prof. Sebastian Raschka, I published a [blog](https://jackli777.github.io/3b1b-some2-2022-/) with an accompanying [3D interactive website](https://jackli777.github.io/visualizing-ai/) based on your published [notebook](https://github.com/rasbt/machine-learning-book/blob/main/ch16/ch16-part1-self-attention.ipynb), to visualize the inner working of Transformer, hope you can check it out!

**Comment by rasbt** [comment]
Wow that's pretty cool! The blog looks really nice. The 3D website doesn't show correctly in my browser though (tried Safari and Chrome. Do you know what's going on?

<img width="1559" alt="Screen Shot 2022-08-11 at 10 39 41 AM" src="https://user-images.githubusercontent.com/5618407/184173450-fffad6ec-c611-4004-8398-31741ec0b955.png">
)

**Comment by jackli777** [comment]
> Wow that's pretty cool! The blog looks really nice. The 3D website doesn't show correctly in my browser though (tried Safari and Chrome. Do you know what's going on?
> 
> <img alt="Screen Shot 2022-08-11 at 10 39 41 AM" width="1559" src="https://user-images.githubusercontent.com/5618407/184173450-fffad6ec-c611-4004-8398-31741ec0b955.png">
> 
> )

Thx! Prof. Sebastian Raschka

In fact, it already shows correctly! You just need to select either one of those 21 scenes, so that cubes will show up!

Just click on the **dropdown menu** on the **control panel** on the **top-right**!
